### PR TITLE
Feature: Issue / TMS links support (fixes #140)

### LINKED ISSUE #140: Add Issues / TMS links support
Currently, it's not possible to add corresponding links to test results report.
Seems like it's not a huge deal according to https://github.com/allure-framework/allure-js-commons/issues/13#issuecomment-218682337
Will raise a PR for that.

### PR DESCRIPTION
**Pull request by sskorol**
- Added common Allure features support: Issues and TMS links.
 - Added new allure.properties resource for testing new features.
 - Updated outdated Allure / WebdriverIO dependencies.

### CODE REVIEW

**Comment by BorisOsipov** [comment]
Hmm looks like travice CI is misconfigured. Need fix it

**Comment by BorisOsipov** [comment]
Fixed. @sskorol I will check

File: lib/reporter.js [hunk 1]
```diff
@@ -358,6 +360,16 @@ class AllureReporter extends events.EventEmitter {
         test.addLabel('severity', severity)
     }
 
+    allureIssue ({ cid, issue }) {
```
Comment (BorisOsipov) [inline lib/reporter.js]: Please, update corresponding README section https://github.com/webdriverio-boneyard/wdio-allure-reporter#supported-allure-api with new methods

**Review by BorisOsipov — APPROVED** [review]
Awesome 👍

File: lib/reporter.js (hunk 1 above)
Comment (sskorol) [reply lib/reporter.js]: oh, forgot it... will do

**Comment by BorisOsipov** [comment]
@sskorol Would you copying this patch to [v5](https://github.com/webdriverio/webdriverio/tree/master/packages/wdio-allure-reporter)?

We will merge once ported to v5

**Comment by sskorol** [comment]
@BorisOsipov so this project will completely migrate to webdriverio since V5?

**Comment by BorisOsipov** [comment]
@sskorol It was already ported to v5. v5 is currently beta. We will support this project some time till users migrate to v5.

**Comment by sskorol** [comment]
@BorisOsipov ok, got it. Will create a PR there as well.

**Comment by BorisOsipov** [comment]
@sskorol Thank you

**Comment by BorisOsipov** [comment]
@christian-bromann 	Can you release new version?

**Comment by christian-bromann** [comment]
Sure

**Comment by christian-bromann** [comment]
wdio-allure-reporter 0.8.0 published 🎉

**Comment by qwez** [comment]
@sskorol hello
How could I provide a link to TMS?
I have created a file `test/resources/allure.properties` with `allure.tests.management.pattern=https://***.testrail.net/index.php?/cases/view/%s`
but my link goes to some local address, like `http://10.0.75.1:50151/3285306`

What can I do?

**Comment by sskorol** [comment]
@qwez I assume you haven't copied `allure.properties` to `allure-results` folder before generating test report.

**Pull request merged**